documentation/tutorials/prerequisites.md

```diff
@@ -31,3 +31,81 @@ $(npm bin)/truffle test --network test
 ```
 
 These tests will take a while to finish, but if set up correctly, all tests should pass.
+
+## Keys and Networks
+
+When using UMA infrastructure, you often have to open the `truffle console` or run a script using `truffle exec`. When
+using truffle in the context of this repository, it's important to understand what types of keys you can use and what
+to specify for the `--network` argument.
+
+### Public Networks
+
+Public networks include the Ethereum mainnet and any public testnets, like Rinkeby, Kovan, or Ropsten. If you are using
+these networks, you'll generally need to know:
+
+1. Which public network you intend to use.
+
+2. What private key you want to use and how your private keys are stored.
+
+The `--network` parameter that's passed to all truffle commands depends on both of these factors. Here's an example:
+
+```bash
+$(npm bin)/truffle console --network rinkeby_mnemonic
+```
+
+That command will tell truffle that the user wants to use the Rinkeby testnet and their private key is a mnemonic, or
+seed phrase. Generally, the network argument is structured as `--network [NETWORK_NAME]_[KEY_TYPE]`. The only exception
+is if you don't provide a key type, it will fall back to a Google Cloud keystore that the UMA team uses internally.
```

Comment:
This bit about falling back to a Goolge Cloud keystore is very confusing and should be cut.

Reply:
Great, I will cut just the last sentence of this, ok? cc @mrice32

Reply:
SGTM.